Makefile

```diff
@@ -65,6 +65,9 @@ test_openstack:
 test_vmware:
 	sudo -E ./tests/test_cli.sh tests/cli/test_build_and_deploy_vmware.sh
 
+clean_cloud_envs:
+	find ./tests/cleanup/ -type f -executable -exec {} \;
```

Comment:
Errr, nope. Why not just list the exact script files we want to execute so it is evident what is going on?